packages/wdio-runner/src/index.js

```diff
@@ -270,26 +270,45 @@ export default class Runner extends EventEmitter {
      * within a hook by the user
      */
     async endSession (shutdown) {
+        /**
+         * For Multiremote - make sure that all the instances have sessionId
+         */
+        const hasSessionId = global.browser && (this.isMultiremote ? !global.browser.instances.some(i => global.browser[i] && !global.browser[i].sessionId) : global.browser.sessionId)
```

Comment:
Pretty long line, perhaps divide over more lines for readability.

Reply:
agree, fixing

Reply:
@erwinheitzman fixed!